### PR TITLE
[API] function to get version

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Add util function to get version of ML-API.

### CODE REVIEW

File: c/src/ml-api-common.c [hunk 1]
```diff
@@ -16,8 +16,33 @@
...
+  if (major)
+    *major = VERSION_MAJOR;
+  if (minor)
+    *minor = VERSION_MINOR;
+  if (micro)
+    *micro = VERSION_MICRO;
+}
+
+/**
+ * @brief Gets the version string of machine-learning API.
```
Comment (songgot) [inline c/src/ml-api-common.c:38]: How about adding  "The returned string should be freed with g_free() when no longer needed." ?

Comment (jaeyun-jung) [reply c/src/ml-api-common.c:38]: I added it in header, not .c.

Comment (songgot) [reply c/src/ml-api-common.c:38]: Thank you. I got it.

**Review by songgot — APPROVED** [review]
LGTM :+1:

**Review by gichan-jang — APPROVED** [review]
LGTM

**Pull request merged**